analysis/src/CompletionFrontEnd.ml

```diff
@@ -587,6 +587,14 @@ let completionWithParser1 ~currentFile ~debug ~offset ~path ~posCursor
         if Debug.verbose () && debugTypedCompletionExpr then
           print_endline "[typedCompletionExpr] No cases, rest";
         ()
+      | Pexp_match (expr, [{pc_lhs; pc_rhs}])
```

Comment:
Perhaps add a comment here as the case is similar to what's in the comment for the next case.